src/main/jbake/content/blog/2017/07/opentracing-ejb.adoc

```diff
@@ -0,0 +1,393 @@
+= OpenTracing EJB instrumentation
+Juraci Paixão Kröhling
+2017-07-28
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger, opentracing, ejb
+
+OpenTracing features more and more framework integrations, allowing for transparent instrumentation
+of applications with minimal effort. This blog post will show how to use the EJB instrumentation
+to automatically trace EJB invocations.
+
+ifndef::env-github[]
+image::/img/blog/2017/2017-07-28-teaser.png[Jaeger with EJB traces]
+endif::[]
+ifdef::env-github[]
+image::../../../../../assets/img/blog/2017/2017-07-28-teaser.png[Jaeger with EJB traces]
+endif::[]
+
+For this demo, we'll generate a project using the Wildfly Swarm project generator, which allows us to
+have a seed project with the appropriate OpenTracing in place. The concrete implementation will be
```

Comment:
"The concrete OpenTracing solution we will use is provided by the [Jaeger project], which ...."